### PR TITLE
chore: update token colors in meter, illustrated message

### PR DESCRIPTION
**Pull request by yihuiliao**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

### CODE REVIEW

File: packages/@react-spectrum/s2/src/IllustratedMessage.tsx [hunk 1]
```diff
@@ -114,9 +114,7 @@ const illustration = style<IllustratedMessageStyleProps & {isInDropZone?: boolea
   '--iconPrimary': {
     type: 'color',
     value: {
-      // TODO: ask design about what the color should be. Says gray-800 in the designs file, neutral in token spec, but different neutral in dropzone spec
-      default: 'gray-800',
-      isInDropZone: 'gray-500', // neutral doesn't seem to match the color in designs, opted for gray-500 instead
+      default: 'neutral',
```
Comment (yihuiliao) [inline packages/@react-spectrum/s2/src/IllustratedMessage.tsx:117]: according to design, the illustration should have the same color both in an IllustratedMessage and in a DropZone. the design files do not currently reflect this but there are tickets to update them

File: packages/@react-spectrum/s2/src/Meter.tsx [hunk 2]
```diff
@@ -82,11 +82,11 @@ const fillStyles = style<MeterStyleProps>({
   borderStyle: 'none',
   borderRadius: 'full',
   backgroundColor: {
-    default: 'informative',
+    default: 'informative-visual',
```
Comment (yihuiliao) [inline packages/@react-spectrum/s2/src/Meter.tsx:85]: matches what is in the design file. confirmed that the token `accent-content-color-default` which is listed in the token file is not correct with spectrum.

**Comment by snowystinger** [comment]
Have we run chromatic against this?

**Comment by yihuiliao** [comment]
link to [chromatic](https://www.chromatic.com/build?appId=5f0dd5ad2b5fc10022a2e320&number=821)

**Review by snowystinger — APPROVED** [review]

**Review by reidbarber — APPROVED** [review]

**Pull request merged**